auto_nag/topcrash.py

```diff
@@ -0,0 +1,283 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from collections import defaultdict
+from datetime import datetime
+from typing import Dict, List, Optional, Set, Union
+
+import libmozdata.socorro as socorro
+from libmozdata import utils as lmdutils
+
+# Top crash identification criteria as defined on Mozilla Wiki.
+#
+# Wiki page: https://wiki.mozilla.org/CrashKill/Topcrash
```

Comment:
I added it to my watchlist, can you do the same so we are notified about changes and we can update the list here accordingly?

Reply:
Good idea! Done.